lesson16/main.js

```diff
@@ -125,6 +127,28 @@ function createNewIconContainer(article) {
   return newIconContainer;
 }
 
+function createCommentIconContainer(article) {
+  const commentIconContainer = document.createElement("div");
+  commentIconContainer.classList.add("comment-container");
+  if (article.comment) {

```

Comment:
IMO
It is possible that the array is empty.
```suggestion
function createCommentIconContainer({article}) {
  const commentIconContainer = document.createElement("div");
  commentIconContainer.classList.add("comment-container");
  if (article.length > 0) {

```